resources/addons/bona-fide-pusher/config.ini

```diff
@@ -0,0 +1,7 @@
+; configuration for REMS bona fide pusher
+[default]
+url = localhost
+port = 3008
+elixir_url = https://perun.elixir-czech.cz/rems/
+elixir_userid = rems-external
+elixir_password = 6kBVNYLcLTs8A5XW
```

Comment:
Should we redact these details from the source or is it OK to publish credentials?